moveit_core/robot_state/include/moveit/robot_state/robot_state.h

```diff
@@ -901,21 +901,13 @@ class RobotState
    */
 
   /**
-   * \brief Convert the frame of reference of the pose to that same frame as the IK solver expects
+   * \brief Transform pose from the robot model's base frame to the reference frame of the IK solver
    * @param pose - the input to change
    * @param solver - a kin solver whose base frame is important to us
    * @return true if no error
    */
   bool setToIKSolverFrame(Eigen::Isometry3d& pose, const kinematics::KinematicsBaseConstPtr& solver);
```

Comment:
Please make this other function private as well. It could become inline also...

Reply:
You mean, make both functions versions `inline`, right?